inc/display-child-pages.php

```diff
@@ -0,0 +1,15 @@
+<?php
+function display_child_pages() {
+	// use shortcode to display in full all direct child pages of current page
```

Comment:
`global $post` can be problematic. Instead perhaps use pre existing native:
`$post_id = get_the_ID();` and then use this var to create the child_pages array

Reply:
Agreed and done